docs/earthfile/earthfile.md

```diff
@@ -1200,6 +1207,35 @@ END
 RUN ./test data # even if this fails, data will have been output
 ```
 
+## CACHE (beta)
+
+{% hint style='info' %}
+##### Note
+The `WAIT` command is experimental and must be enabled via `VERSION --wait-block 0.6`.
```

Comment:
Should this say The `CACHE` command is experimental?

Reply:
Oh yeah, good catch. I'll update.